lldb/source/Plugins/Language/Swift/SwiftLanguage.cpp

```diff
@@ -1894,6 +1894,86 @@ SwiftLanguage::GetDemangledFunctionNameWithoutArguments(Mangled mangled) const {
   return mangled_name;
 }
 
+static std::optional<llvm::StringRef>
+GetDemangledBasename(const SymbolContext &sc) {
+  return std::nullopt;
+}
+
+static std::optional<llvm::StringRef>
+GetDemangledFunctionPrefix(const SymbolContext &sc) {
+  return std::nullopt;
+}
+
+static std::optional<llvm::StringRef>
+GetDemangledFunctionSuffix(const SymbolContext &sc) {
+  return std::nullopt;
+}
+
+static bool PrintDemangledArgumentList(Stream &s, const SymbolContext &sc) {
+  return false;
+}
```

Comment:
Did you mean to have these unused functions as part of this patch? Should `HandleFrameFormatVariable` call these?

Reply:
You are right, my initial goal was to only add the skeleton of the function but `HandleFrameFormatVariable` should at least call them, otherwise they are truly unused. I will fix that.